feature/login/src/main/java/com/iteneum/login/presentation/LoginView.kt

```diff
@@ -0,0 +1,182 @@
+package com.iteneum.login.presentation
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.Dimension
+import com.iteneum.designsystem.components.LpFilledTonalButton
+import com.iteneum.designsystem.components.LpOutlinedTextFieldMail
+import com.iteneum.designsystem.components.LpOutlinedTextFieldPassword
+import com.iteneum.designsystem.R.drawable.*
+import com.iteneum.login.R
+
+
+@Composable
+fun LoginView() {
+    ConstraintLayout(
+        modifier = Modifier
+            .fillMaxSize()
+
+    ) {
+        val (logo, mail, password, tonalButton, loginWith, socialNetworkLogins, registerButton) = createRefs()
+
+        Image(
+            painter = painterResource(id = leasepertlogo),
```

Comment:
Please change the logo name, ask to Irving if it should be leasepertLogo or leasepert_logo. Sorry I don't remember